test/integration/setup.js

```diff
@@ -302,14 +308,14 @@ async function setup() {
       },
       {
         productType: 2, // Yield Token Cover
-        ytcUnderlyingAsset: '0x0000000000000000000000000000000000000001',
+        ytcUnderlyingAsset: ybETH.address,
```

Comment:
I don't think this is the correct asset to be used here. It should be the underlying asset, not the covered one.

Reply:
It is right, it's the covered asset.

There was a rename to this new name

https://github.com/NexusMutual/smart-contracts/blob/490c3a93cd888b6d34083be27df0fdd6ad860e12/contracts/modules/assessment/YieldTokenIncidents.sol#L294